docker-container/launch-m1.sh

```diff
@@ -0,0 +1,28 @@
+source config.sh
+set -x
+# Clean environment from before
+rm -rf docker-magento
+
+python set-magento-version.py $m1Version
+docker-compose down -v
+docker-compose up -d
+sleep 5
+
+# Install Sample Data and Initialize Magento
+if [ "$installSampleData" = true ] ; then
+    docker exec -it magento1_store install-sampledata
+fi
+sleep 1
+docker exec -it magento1_store install-magento
+
+# Installs Bolt and sets the keys for it
+if [ "$localRepo" = true ] ; then
+    docker cp $boltRepo magento1_store:/var/www/html/
+else
+    git clone -b $boltBranch git@github.com:BoltApp/bolt-magento1.git
+    docker cp ./bolt-magento1 magento1_store:/var/www/html/
+    rm -rf ./bolt-magento1
+fi
+docker exec magento1_store bash -c "cp -r /var/www/html/bolt-magento1/* /var/www/html/"
```

Comment:
maybe you can inline this in above if/else and avoid copying files twice when $localCopy is true?

Reply:
this is needed for both cases, just tested to confirm